src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestMethodCommand.class.st

```diff
@@ -0,0 +1,98 @@
+"
+Description
+--------------------
+
+I am a command available in the method context menu of Calypso. When activated I'll jump on the test method corresponding to the method. 
+
+If the method does not exists I will create it in the corresponding class.
+
+Internal Representation and Key Implementation Points.
+--------------------
+
+    Instance Variables
+	selectedMethods:			<aCollection>	Methods selected by the user that should have their tests generated.
+	testMethodToBrowse:		<aMethod>			The method to browse at the end of the execution of the command.
+
+"
+Class {
+	#name : #ClyJumpToTestMethodCommand,
+	#superclass : #CmdCommand,
+	#traits : 'TClyGenerateTestClass',
+	#classTraits : 'TClyGenerateTestClass classTrait',
+	#instVars : [
+		'selectedMethods',
+		'testMethodToBrowse'
+	],
+	#category : #'Calypso-SystemPlugins-SUnit-Browser-Commands'
+}
+
+{ #category : #testing }
+ClyJumpToTestMethodCommand class >> canBeExecutedInContext: aBrowserContext [
+	^ aBrowserContext isInstanceSideMethodSelected and: [ aBrowserContext selectedMethods anySatisfy: [ :each | each isTestMethod not ] ]
```

Comment:
Why instance side is really important?

Reply:
It was the behavior in Pharo 6. If you applied the command on the class side it did nothing.

Maybe we can improve the behavior later but I wanted to at least have what we had in Nautilus.